packages/maskbook/src/extension/popups/pages/Wallet/ContractInteraction/index.tsx

```diff
@@ -240,11 +238,8 @@ const ContractInteraction = memo(() => {
     const tokenDecimals = (isNativeTokenInteraction ? nativeToken?.decimals : token?.decimals) ?? 0
 
     // token estimated value
-    const tokenPrice = useTokenPrice(
-        getCoingeckoPlatformId(chainId) ?? '',
-        !isNativeTokenInteraction ? token?.address : undefined,
-    )
-    const nativeTokenPrice = useNativeTokenPrice(getCoingeckoCoinId(nativeToken?.symbol.toLowerCase() ?? '') ?? '')
+    const tokenPrice = useTokenPrice(chainId, !isNativeTokenInteraction ? token?.address : undefined)
+    const nativeTokenPrice = useNativeTokenPrice(nativeToken?.chainId ?? 0)
```

Comment:
`0` is not a valid chain id. 😔

Reply:
Then will fail to get valid platformId/coinId, and price will be `0`